3ds Max/Max2Babylon/Exporter/BabylonExporter.IMaterialExporter.cs

```diff
@@ -35,7 +77,7 @@ internal interface IGLTFMaterialExporter : IMaterialExporter
         /// <param name="raiseWarningAction">Callback function to raise warnings.</param>
         /// <param name="raiseErrorAction">Callback function to raise errors.</param>
         /// <returns>The exported GLTF material.</returns>
-        GLTFMaterial ExportGLTFMaterial(BabylonExporter exporter, GLTF gltf, IIGameMaterial material, Func<string, string, string> tryWriteImageFunc,
+        GLTFMaterial ExportGLTFMaterial(ExportParameters exportParameters, GLTF gltf, IIGameMaterial material, Func<string, string, string> tryWriteImageFunc,
```

Comment:
@elpie89 , i wanted to change this interface, i'm not sure if you'd really need the BabylonExporter as a whole to export the MaxMaterial correctly, but since this is a breaking change, let me know if we should have an alternative.

Reply:
@Selmar worked on this interface from the beginning, I didn't touch the IMaterialExporter...he should know better than me
What do you think @Selmar ?

Reply:
Looking at the current code, the only thing we need are the export parameters. This should be fine.

I haven't suggested this before, but I feel the material exportation in the babylon exporter could make use of this same interface, as opposed to having all the exports in one big function as it is now. It should not be very difficult, just need to know the ClassID, implement the interface and move the code, although some parameters may need to be added/changed (I haven't looked at this since I made it).

Reply:
Of course, as it isn't used by anyone, the IBabylonMaterialExporter is currently not complete in any way.

Reply:
Sounds good, thanks for the input 👍